### DISCUSSION TITLE
Url '/' defines a query parameter 'id' that is missing in exportPathMap

### DISCUSSION THREAD

**Discussion by connor-wantomo** [vercel/next.js#22330] [Help]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

14.15.5

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**How are you deploying your application?**

next export, aws amplify

**Describe the Bug**

I'm using next export alongside exportPathMap to map all the pages, this does work fine. However, when adding a query to the URL I get this error in my console:

Url '/' defines a query parameter 'id' that is missing in exportPathMap

I think this error is causing issues when I'm running queries on load, maybe because the id query isn't found inside the router object.

**Expected Behavior**

Router to pass the id query on page load

**To Reproduce**

Next export, create a exportPathMap and enter url query (/?id=101)

**Comment by cha2hyun** [comment]
+1

**Reply by cha2hyun** [reply]
Any updates?
